enzyme/test/Enzyme/ForwardMode/blas/cblas_ddot.ll

```diff
@@ -44,6 +44,7 @@ entry:
   ret double 1.0
 }
 
+; XFAIL: *
```

Comment:
You should make one test with runtime activity, and one test without (rather than this single test).